tests/framework/e2e/cluster.go

```diff
@@ -152,18 +152,18 @@ type EtcdProcessClusterConfig struct {
 	Client        ClientConfig
 	IsPeerTLS     bool
 	IsPeerAutoTLS bool
-	NoCN          bool
+	CN            bool
 
 	CipherSuites []string
 
-	ForceNewCluster            bool
-	InitialToken               string
-	QuotaBackendBytes          int64
-	DisableStrictReconfigCheck bool
```

Comment:
No sure we should change it.

Both Disable or No seems to enable it by default. If we switch to `CN`,  I think we should change all the places to `DefaultXXXConfig`, like [DefaultConfig][1]. What do you think?

[1]: <https://github.com/etcd-io/etcd/blob/ddd65e7fe2e2d6fbb9a89a07ddbef79412885659/tests/framework/e2e/cluster.go#L187>

Reply:
Looks like DefaultConfig was updated. As this is just an internal test framework I don't see any negative impacts of changing the defaults.

Reply:
My bad. The `WithConfigXYZ` always call `DefaultConfig`. For sure, it looks good.

Reply:
Thanks for taking the time to review, I should have explained it earlier😬